Source/WebCore/platform/graphics/mac/controls/SwitchTrackMac.mm

```diff
@@ -190,6 +192,12 @@
         trackImage->context().drawConsumingImageBuffer(WTFMove(toImage), IntPoint(), ImagePaintingOptions { CompositeOperator::PlusLighter });
     }
     context.drawConsumingImageBuffer(WTFMove(trackImage), inflatedTrackRect.location());
+
+    if (isFocused) {
+        // FIXME: This does not incorporate the style data as ControlMac::drawCellFocusRing() does.
```

Comment:
I don't think this `FIXME` is needed? I'm not aware of any style data we would incorporate here, since the things in `drawCellFocusRing` are specific to `NSCell` configuration.

Reply:
Okay, I can remove it. I wasn't sure if there were any settings we would not be taking into consideration this way.